index.js

```diff
@@ -277,6 +277,7 @@ export async function main () {
   if (args.payload) {
     for await (const result of push(args)) {
```

Comment:
The loop here is unnecessary. Can be replaced with something like this:
```js
const result = (await push(args).next()).value
debugResult(result)
return
```

Reply:
Fixed it so the `-p` flag can be provided multiple times.